hawc/apps/assessment/views.py

```diff
@@ -365,6 +366,8 @@ def get(self, request, *args, **kwargs):
             raise PermissionDenied()
 
         assessment.bust_cache()
+        refresh_views_by_cache()
```

Comment:
outside the materialized model, i'd be confused by the meaning of views.  Maybe rename to `update_all_mvs()`.

Also, this would only work if our refresh flag is set I think right? It'd be better if this method would just refresh all mvs, no matter what.

Reply:
I believe that calling a celery task as a function does it synchronously, based on [this stackoverflow answer](https://stackoverflow.com/a/55537707) and the following from the [celery docs](https://docs.celeryproject.org/en/stable/userguide/calling.html):
```
calling (__call__)
    Applying an object supporting the calling API (e.g., add(2, 2)) means that
    the task will not be executed by a worker, but in the current process instead
    (a message won’t be sent).
```
If you want to keep synchronous functions separate from our celery tasks though I don't mind just making a separate function that the task calls instead.

Reply:
Thanks, this is good to know. 

The way I generally view it, celery tasks are designed more just to implement how we run something, but shouldn't be the only place that contain the business logic. This allows the business logic to be tested and checked or used in other ways w/o the celery infrastructure.  We wont easily be able to write tests for this specific task, but thats my general mindset as to why code is in different places.

Given that, I'd prefer a separate function that the task and other things could call.  Would you agree?

Reply:
Yep, I agree with that approach.